### PR TITLE
Feature/milestone 5

### PR DESCRIPTION
**Pull request by jpdf00**
- Style all components according to the guide provided on Zeplin.

### CODE REVIEW

File: src/assets/stylesheets/BooksForm.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  background-color: #0290ff;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+}
```
Comment (rindrajosia) [inline src/assets/stylesheets/BooksForm.css:68]: no repeating of the same rule

File: src/assets/stylesheets/BooksList.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  background-color: #0290ff;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+}
```
Comment (rindrajosia) [inline src/assets/stylesheets/BooksList.css]: no repeating of the same rule

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- All element in the design is added :+1: 
- Styling is **EXACTLY** the same as the one provided in the designs :100: 

To improve :factory_worker: [REQUIRED]:

- [x] Please make sure that CSS bad practices are not used  ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

In this screenshot, for example, you have the same rule for `submitButton` inside `BooksForm.css` and `BooksList.css`. I suggest that you create a file where you put this rule then both components will use just this file.

Nice to have [OPTIONAL] :

- [ ] 

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

File: src/assets/stylesheets/BooksList.css (hunk 2 above)
Comment (jpdf00) [reply src/assets/stylesheets/BooksList.css]: Thanks for the suggestion and for spotting this mistake. The rule in BooksList is actually unnecessary so it was removed.

**Review by afanimartin — APPROVED** [review]
## Approved 🙏🏿 🙏🏿 🙏🏿 
Nice work making the requested changes. There's no repeat of styling. Well done.


### Martin Afani
#### Slack: whiz25

**Pull request merged**